mods/persistence/modules/organs/internal/stack.dm

```diff
@@ -43,6 +42,11 @@ GLOBAL_LIST_EMPTY(corticalStacks)
 	var/prompting = FALSE // Are we waiting for a user prompt?
 	var/mob/stack/stackmob = null
 
+/obj/item/organ/internal/stack/Initialize(var/mapload, var/datum/dna/given_dna)
+	. = ..()
+	
+	new /obj/item/radio/stack_radio(src)
```

Comment:
Add a buildpath to the stack and have this spawn as a component instead.

Reply:
Not sure what you mean exactly. There's not any supporting mechanics for producing organs with components as far as I know.